tool/jt.rb

```diff
@@ -838,17 +847,20 @@ def test_cexts(*args)
     [
         ['oily_png', ['chunky_png-1.3.6', 'oily_png-1.2.0'], ['oily_png']],
         ['psd_native', ['chunky_png-1.3.6', 'oily_png-1.2.0', 'bindata-2.3.1', 'hashie-3.4.4', 'psd-enginedata-1.1.1', 'psd-2.1.2', 'psd_native-1.1.3'], ['oily_png', 'psd_native']],
-        ['nokogiri', [], ['nokogiri']]
-    ].each do |gem_name, dependencies, libs|
+        ['nokogiri', [], ['nokogiri']],
+        ['ruby-argon2', [], ['ruby-argon2'], {'GEM_HOME' => "#{ENV['GEM_HOME']}/bundler/gems/ruby-argon2-afe6d65e37c6", 'TEST_CHECKS' => '1'}]
```

Comment:
We need a way to specify the path to the argon2 dir in the test pack though, probably can be an extra argument in this list.